lib/internal/timers.js

```diff
@@ -342,13 +342,33 @@ function insert(item, refed, start) {
   L.append(list, item);
 }
 
-function setUnrefTimeout(callback, after) {
+function setUnrefTimeout(callback, after, arg1, arg2, arg3) {
   // Type checking identical to setTimeout()
   if (typeof callback !== 'function') {
     throw new ERR_INVALID_CALLBACK(callback);
   }
 
-  const timer = new Timeout(callback, after, undefined, false);
+  var i, args;
+  switch (arguments.length) {
+    // fast cases
+    case 1:
+    case 2:
+      break;
+    case 3:
+      args = [arg1];
+      break;
+    case 4:
+      args = [arg1, arg2];
+      break;
+    default:
+      args = [arg1, arg2, arg3];
```

Comment:
Nit: why not moving this to its own case and run the loop only when `arguments.length > 5`?

Reply:
This matches exactly what we do for `setTimeout`. Wanted them to be consistent.

Reply:
I am -1 on adding this. We do not use any of these arguments and this function is only used internally. I removed that code recently for that reason: #26555

Reply:
It does seem as if we do not use any of the args anywhere. @apapirovski Are you OK removing the switch statement? (Or am I missing something?)

Reply:
Ping @apapirovski